x/dsmr/block.go

```diff
@@ -119,3 +127,28 @@ type Block struct {
 func (b Block) GetID() ids.ID {
 	return b.blkID
 }
+
+func (b Block) Parent() ids.ID {
+	return b.ParentID
+}
+
+func (b Block) Timestamp() int64 {
+	return b.Tmstmp
+}
+
+func (b Block) Height() uint64 {
+	return b.Hght
+}
+
+func (b Block) Txs() []*ChunkCertificate {
+	return b.ChunkCerts
+}
```

Comment:
This seems weird because these are not returning txs

Reply:
yes, I agree. let's discuss this as a group ?

Reply:
Can we use the term containers?